pkg/cmd/version/version.go

```diff
@@ -35,18 +37,27 @@ const devVersion = "dev"
 const latestReleaseURL = "https://api.github.com/repos/tektoncd/cli/releases/latest"
 
 // Command returns version command
-func Command() *cobra.Command {
+func Command(p cli.Params) *cobra.Command {
 	var check bool
 
 	var cmd = &cobra.Command{
 		Use:   "version",
-		Short: "Prints version information",
+		Short: "Prints tkn and server version information",
 		Annotations: map[string]string{
 			"commandType": "utility",
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
 			fmt.Fprintf(cmd.OutOrStdout(), "Client version: %s\n", clientVersion)
 
+			cs, err := p.Clients()
+			if err == nil {
+				version, _ := version.GetPipelineVersion(cs)
+				if version == "" {
+					version = "unknown"
+				}
+				fmt.Fprintf(cmd.OutOrStdout(), "Server version: %s\n", version)
```

Comment:
Do we maybe say Pipeline version as opposed to Server version?

Reply:
maybe, I am not so sure.. kubectl version will show the the same and talk about the core engine version not the operators. but either way i don't mind ...

Reply:
Think it would be useful if we chose to also show Triggers version information, but cool with Server if not.

Reply:
Okay, i changed it to Pipeline